ee/codegen/src/__test__/helpers/node-data-factories.ts

```diff
@@ -650,33 +653,63 @@ export function templatingNodeFactory({
   errorOutputId?: string;
   inputRules?: NodeInputValuePointerRule[];
   outputType?: VellumVariableType;
+  inputReferenceId?: string;
+  inputReferenceNodeId?: string;
+  template?: NodeInputValuePointerRule;
 } = {}): TemplatingNode {
+  const defaultTemplate: NodeInputValuePointerRule = {
+    type: "CONSTANT_VALUE",
+    data: {
+      type: "STRING",
+      value: "Hello, World!",
+    },
+  };
+
   const nodeData: TemplatingNode = {
-    id: id ?? "7e09927b-6d6f-4829-92c9-54e66bdcaf80",
+    id: id ?? "46e221ab-a749-41a2-9242-b1f5bf31f3a5",
     type: WorkflowNodeType.TEMPLATING,
     data: {
       label: label ?? "Templating Node",
       outputId: "2d4f1826-de75-499a-8f84-0a690c8136ad",
       errorOutputId,
-      sourceHandleId: sourceHandleId ?? "dd8397b1-5a41-4fa0-8c24-e5dffee4fb98",
-      targetHandleId: targetHandleId ?? "3feb7e71-ec63-4d58-82ba-c3df829a2948",
+      sourceHandleId: sourceHandleId ?? "6ee2c814-d0a5-4ec9-83b6-45156e2f22c4",
+      targetHandleId: targetHandleId ?? "3960c8e1-9baa-4b9c-991d-e399d16a45aa",
```

Comment:
[nit] would prefer if we reverted these too so that we minimize blast radius on other tests as aggressively as possible